mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -18,6 +18,7 @@
 
 <div class="list-container">
 	<div class="header">
+
```

Comment:
Two base rules of formatting HTML, as mentioned before:
1. Child node should indented by tab, no blank row.
1. Sibling nodes should be separated by a blank row.

It was fine as it was.